### DISCUSSION TITLE
Presenting stdlib at FortranCon 2021

### DISCUSSION THREAD

**Discussion by awvwgk** [fortran-lang/stdlib#461] [Ideas]
For FortranCon 2021 we want to present the core projects of the Fortran-lang community again. Here I want to organize the presentation(s) for the stdlib project. But first, we are looking for a volunteer who wants to lead the presentation of stdlib and give the talk at FortranCon.

Also, for our two GSoC students (@Aman-Godara and @ChetanKarwa) working on stdlib, we are planning to organize separate talks as well.

---

You can checkout the presentation of stdlib from last year held by @jvdp1 [here](https://youtu.be/qIZ0woAZ28k)

FortranCon 2021 announcement: https://fortran-lang.discourse.group/t/1485

**Comment by milancurcic** [comment]
I suggest @ivan-pi, @nshaffer, @hsnyder, or @jvdp1 as significant contributors who are more or less early career. Though @jvdp1 did a great job last year, I think it'd be nice to rotate speakers so that we give more exposure to as many people as possible.

**Reply by ivan-pi** [reply]
Thanks @milancurcic, I'd be very happy to take the lead on this if no else wants to volunteer. Sadly, I was not very active this year with code contributions. 

The bulk of the work achieved has been thanks to @wclodius2, @awvwgk, @jvdp1, and several new contributors including @hsnyder, @ejovo13, @ghbrown, @ChetanKarwa, @Aman-Godara, among others.

I had some plans to participate in FortranCon with a talk on Design of ODE Solver interfaces, but its still on the drawing table. @awvwgk, are you also planning to present some of your others projects?

Here is a an overview of the new content in stdlib compared to last year:
- bitsets
- sorting
- logger
- string type and associated methods
- legendre polynomials and Gauss quadrature
- clip, outer_product, and soon linspace
- improvements to the build system and documentation

**Reply by hsnyder** [reply]
Thanks for tagging me. I'm happy to help put together the presentation in whatever way I can, but I still feel fairly new to stdlib - Ivan is probably a better choice than myself to take the lead.

**Reply by jvdp1** [reply]
I agree with @milancurcic: it would be good to rotate speakers.Anyway, I will not be able to join FortranCon 2021 (or only partially), due to other meetings already scheduled on these days.

The list of contributors mentioned by @ivan-pi is appropriate. Should we agree now on a speaker? We can also discuss it during the next Fortran month call.

**Reply by ghbrown** [reply]
I am in the same boat as @hsnyder. Very happy to help with what I can, and perhaps after a few more substantial contributions and another year I'd be happy to speak if needed.

**Reply by nshaffer** [reply]
I think I could do this. I haven't been able to contribute much to stdlib this year, but that doesn't seem like a show-stopper. If anything, it'll help to keep the focus at a high level. Plus, it's nice to talk about other people's accomplishments. And if my volunteering increases the chances that @ivan-pi will tell us about ODE solvers, all the better.

**Reply by ivan-pi** [reply]
In this case @nshaffer please go ahead. 

The next step will be to add a new folder "FortranCon2021-stdlib" in https://github.com/fortran-lang/talks, and decide to go for latex/beamer, Reveal.js, or something else. The deadline for abstracts is August 1.

I will be happy to contribute to the presentation with some usage examples of bitsets if space/timeslot permit.

**Comment by ejovo13** [comment]
Ditto!    On Tuesday, July 13, 2021, 02:47:46 PM EDT, Gabriel Brown ***@***.***> wrote:  
 
 


I am in the same boat as @hsnyder. Very happy to help with what I can, and perhaps after a few more substantial contributions and another year I'd be happy to speak if needed.

**Comment by arjenmarkus** [comment]
I'd love to hear about the ODE solvers :).

Op wo 14 jul. 2021 om 02:43 schreef Evan Voyles ***@***.***>:

>
> Ditto! On Tuesday, July 13, 2021, 02:47:46 PM EDT, Gabriel Brown

**Comment by nshaffer** [comment]
OK, I will take the lead on this. By early next week, I will have a new directory in the talks repo and a first draft of an abstract. I'll allow a week to collect feedback and get the author list straightened out. That should give plenty of time to make revisions (and accept late feedback 😉 ) before the submission deadline.

**Comment by nshaffer** [comment]
Hi all, I have put up a first draft and tentative author list as a PR in the talks repo https://github.com/fortran-lang/talks/pull/10

Let's continue the conversation there!
